Tests/MeiliSearchIntegrationTests/UpdatesTests.swift

```diff
@@ -72,7 +72,7 @@ class UpdatesTests: XCTestCase {
 
                     switch result {
                     case .success(let update):
-                        XCTAssertTrue(["enqueued", "processed", "fail"].contains(update.status))
+                        break
```

Comment:
Where is the test? 🙂

Reply:
the test is the type itself, if MeiliSearch returns anything different from the values `ResultType`, the decode will fail. I believe you guys aren't planning to include any new status in the version 0.15, don't you?

Reply:
Nope, but the status names will change into `in_process`, `failed`, and `done`, see https://github.com/meilisearch/MeiliSearch/issues/998.

Reply:
I will rename it to the new values and add support to the old ones. What about it?

Reply:
Rename them in the future, yes. No need to support the old ones 😉